share/lib/python/neuron/hclass2.py

```diff
@@ -4,12 +4,12 @@
 # h.anyclass methods may be overridden. If so the base method can be called
 # using the idiom self.basemethod = self.baseattr('methodname')
 # ------------------------------------------------------------------------------
-      
+
 from neuron import h, hoc
 import nrn
 
 #avoid syntax error if compiled by python 3
```

Comment:
no need for this comment anymore. Or perhaps modify the the fact that it would be a syntax error but will not be imported by python3.

Reply:
nevermind. No changes needed.